appcelerator/titanium_mobile
# chore(android): update "ti.playservice" module to 17.5.0

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-27934

**Summary:**
- Updated Google Play Services library from `17.1.1` to `17.5.0`.
- Changed `makeGooglePlayServicesAvailable()` method to use native implementation.
  * Now shows Google's standard dialog if Play Services needs to be updated.
  * Couldn't use native implementation before because backing out of the activity that was hosting this dialog used to crash on Google's end. Google has now fixed it.
- Updated to support upcoming "run-in-background" feature in the future. See: [TIMOB-27896](https://jira.appcelerator.org/browse/TIMOB-27896)

**Test:**
1. Open the Android emulator.
2. Go to: Settings -> Apps -> Google Play Services
3. Tap on the "Disable" button.
4. Build and run [kitchen-sink-v2](https://github.com/appcelerator/kitchensink-v2).
5. On app startup, verify a dialog appears asking you to enable Google Play Services.
6. Tap on the dialog's okay button.
7. Tap on the "Back" button. _(Do **NOT** tap on the dialog's okay button.)_
8. Verify you are exited out of the app.
9. Re-launch the app.
10. On app startup, this time tap on the dialog's okay button.
11. Verify that you are taken to the Google Play Services "App Info" screen.
12. Do not enable it. Instead, tap on the "Back" button.
13. Verify you are exited out of the app.
14. Re-launch the app.
15. Tap on the dialog's okay button.
16. Tap on the "Enable" button in the Google Play Services window.
17. Tap on the "Back" button.
18. Verify the kitchen sink app now launches.
19. Back out of the app.
20. Re-launch the app.
21. Verify the Google Play Services dialog does **NOT** appear and you are taken directly to the app.
